mgmt_cli_test.py

```diff
@@ -77,6 +78,14 @@ def _get_cluster_hosts_with_ips(self):
             Setup.INTRA_NODE_COMM_PUBLIC else 'private_ip_address'
         return [[node, getattr(node, ip_addr_attr)] for node in self.db_cluster.nodes]
 
+    def get_all_dcs_names(self):
+        regions_names = set()
```

Comment:
the name "regions_names" may be confusing since deals with 'data_center' and the function name is "get_all_dcs_names". please consider changing to "data_center_names".